fredericoo/saltplay
# fix: season 404 error

# 👀 Overview

Annoying error when trying to load a season page is now poof.

## 🚀 What we have done

- Removed pre-built routes (getStaticPaths) so they're not built on deployment, but upon request
- Added `revalidate` even to when 404 page hits (that's the main bug, if it returns 404 it stops revalidating the page so even if the page is created later, it'll need a rebuild)
- Fixed `signin` error adding noise to our logs

## 🏁 How to test

- Load up a season page
- Change the season slug on your address bar (say, from `season-1` to `season-2`)
- It should 404
- Now create the season with the slug that you tried entering
- Visit its page
- It should either load, or 404
- Refresh the page
- It should load

## ✅ Checklist:

- I have created automated tests to replicate my manual testing automatically
